executor/benchmark_test.go

```diff
@@ -599,6 +599,98 @@ func prepare4HashJoin(testCase *hashJoinTestCase, innerExec, outerExec Executor)
 	return e
 }
 
+// Note the innerExec and outerExec are reversed, this is different from prepare4Join
+func prepare4OuterJoin(testCase *hashJoinTestCase, innerExec, outerExec Executor) *HashJoinExec {
```

Comment:
Could we reuse `prepare4HashJoin`?

Reply:
No, due to some different attributes, like `joinType`, `useOuterToBuild`.

Reply:
already merge some functions.